pkgs/by-name/ja/jailer/package.nix

```
@@ -0,0 +1,82 @@
+{
+  copyDesktopItems,
+  fetchFromGitHub,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  stdenv,
+  wrapGAppsHook4,
+  ant,
+  jdk,
+  jre,
+  stripJavaArchivesHook,
+}:
+stdenv.mkDerivation rec {
```

[Use `finalAttrs` instead of `rec`](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)
```suggestion
stdenv.mkDerivation (finalAttrs: {
```